src/backend/vulkan/src/physical_device.rs

```diff
@@ -1287,6 +1426,7 @@ impl adapter::PhysicalDevice<Backend> for PhysicalDevice {
             performance_caveats: Default::default(),
             dynamic_pipeline_states: DynamicStates::all(),
             downlevel: DownlevelProperties::all_enabled(),
+            external_memory_limits: external_memory_limits,
```

Comment:
nit: `external_memory_limits,`

Reply:
Done. I have found other instances where i made the same error, corrected them too.